packages/leavitt-elements/src/leavitt-company-select.ts

```diff
@@ -0,0 +1,209 @@
+import { css, html, LitElement } from 'lit';
+import { property, customElement, query } from 'lit/decorators.js';
+import '@material/mwc-select';
+import '@material/mwc-list/mwc-list-item.js';
+
+import { Select } from '@material/mwc-select';
+
+import { LoadWhile } from '@leavittsoftware/titanium-helpers';
+import { Company } from '@leavittsoftware/lg-core-typescript/lg.net.core';
+import ApiService from '@leavittsoftware/api-service/lib/api-service';
+import { DOMEvent } from './dom-event';
+import { TitaniumSnackbarSingleton } from '@leavittsoftware/titanium-snackbar/lib/titanium-snackbar';
+
+/**
+ *  Single select input that searches Leavitt Group companies
+ *
+ *  @element leavitt-company-select
+ *
+ *  @fires selected - Fired when a selection has been made. Property index is the selected index (will be of type number)
+ *  @fires opened - Fired menu opens
+ *  @fires closed - Fired menu closes
+ *
+ */
+@customElement('leavitt-company-select')
+export class LeavittCompanySelectElement extends LoadWhile(LitElement) {
+  @query('mwc-select') protected select: Select & { mdcFoundation: { setValid(): boolean }; isUiValid: boolean };
+
+  /**
+   *  Required if you want component to load companies automatically
+   */
+  @property({ attribute: false }) apiService: ApiService | null;
+
+  /**
+   *  If no service is supplied, consumer can bind in custom list of companies
+   */
+  @property({ type: Array }) companies: Array<Partial<Company>> = [];
+
+  /**
+   *  The person object selected by the user.
+   */
+  @property({ type: Object }) value: Partial<Company> | null = null;
+
+  /**
+   *  Message to show in the error color when the element is invalid.
+   */
+  @property({ type: String }) validationMessage: string;
+
+  /**
+   *  Sets floating label value.
+   */
+  @property({ type: String }) label: string = 'Company';
+
+  /**
+   *  Helper text to display below the select. Always displays by default.
+   */
+  @property({ type: String }) helper: string = '';
+
+  /**
+   *  Whether or not the input should be disabled.
+   */
+  @property({ type: Boolean }) disabled: boolean = false;
+
+  /**
+   *  Sets the dropdown menu's position to fixed. This is useful when the select is inside of a stacking context e.g. inside of an mwc-dialog. Note, that --mdc-menu-min-width or --mdc-menu-max-width may have to be set to resize the menu to the width anchor.
+   */
+  @property({ type: Boolean }) fixedMenuPosition: boolean = true;
+
+  /**
+   *  Displays error state if no image is empty and input is blurred.
```

Comment:
Has the same description as validityTransform, I think they were both copied from image select